R/nest-string.R

```diff
@@ -0,0 +1,93 @@
+#' Create a nested column.
+#'
+#' Collapse a column's values into rows of delimited strings defined by the
+#' the remaining variables.
+#'
+#' @inheritParams nest_string_
+#' @seealso \code{\link{unnest_string}} for the converse function
+#' @export
+#' @examples
+#'
+#' df <- data.frame(
+#'   x = c(1, 2, 2, 2, 3, 3),
+#'   y = c("a", "d", "e", "f", "g", "h"),
+#'   stringsAsFactors = FALSE
+#' )
+#' nest_string(df, y)
+nest_string <- function(data, col = NULL, sep = ",") {
```

Comment:
I'm not sure I want `nest_string()` - I think `unnest_string()` is useful, but the reverse seems a little ill-specified to me (esp how the grouping work)